### ISSUE TITLE
.container and other Styles

### ISSUE THREAD

**Issue by Rvolume** [marcosmoura/vue-material#289]
Didn't find related issues, sorry if there is & sorry if question seems to be stupid.

Can you, please, tell me - how do you see CSS usage?

My problem is:
 - atm I need something like .container class from bootstrap / materialcss, but there's not much CSS grid-alike classes at vuematerial, most css docs are for concrete components.
 - MAYBE in a future I'll need something else from materialcss styles, which you not implemented (yet?)

So I'm not sure what to do better - import non-native materialcss styles to extend vuematerial styles or wait for new versions? Or you don't see any need/concept in those .containers for vuematerial at all & I should simulate container with md-layouts/offsets?

In summary: is it ok to import materialcss styles aside vuematerial or that is a thing everyone do already?

**Comment by korylprince** [comment]
Have you seen the layout section of the docs?

https://vuematerial.github.io/releases/v0.5.1/#/ui-elements/layout

**Comment by Rvolume** [comment]
Thanks for response. Yes, i saw it (but v 5.1 has working demos, surprisingly).
So the answr is - use md flex + offsets, no familiar containers.
Thanks!
